toolbox/CMakeLists.txt

```diff
@@ -287,6 +301,14 @@ if(WBT_USES_ICUB)
 #    endif()
 endif()
 
+if(WBT_USES_QPOASES)
+    target_compile_definitions(WBToolboxLibrary PUBLIC "WBT_USES_QPOASES")
+    target_link_libraries(WBToolboxLibrary PRIVATE ${qpOASES_LIBRARIES})
+
+    # Manually include qpOASES include folders. They are not exported by the linked library.
+    include_directories(${qpOASES_INCLUDE_DIRS})
```

Comment:
You can also use `target_include_directories(WBToolboxLibrary PRIVATE ${qpOASES_INCLUDE_DIRS})` .